### ISSUE TITLE
Recommendations for enabling system service should be changed for non-systemd setups

### ISSUE THREAD

**Issue by jamalex** [learningequality/kolibri-installer-debian#75]
If you choose not to start on boot, it gives these instructions for changing your mind later:

https://github.com/learningequality/kolibri-installer-debian/blob/4c3c6ff83fc3995fc337ef0dd736feca2402b61a/debian/kolibri.templates#L36

However, these don't work on Ubuntu 14.04 and other non-systemd setups. Instead, for these, it should recommend `update-rc.d kolibri defaults`.

**Comment by benjaoming** [comment]
We could write an alternative instruction template and display it if `which systemctl` fails.

**Comment by jredrejo** [comment]
yes, code to detect if `systemd` is active or not is already in the postisnt, so it's just a question of using it as a function to show different templates

**Comment by jredrejo** [comment]
@jamalex taking into account we don't support officialy Ubuntu 14.04:
https://kolibri.readthedocs.io/en/latest/install.html#debian-ubuntu
I think we can close this issue.

**Comment by benjaoming** [comment]
:+1:  Unfortunately only very few distributions are working with alternatives to SystemD, and they all have very enlightened users that will understand what's going on :)
